app/models.py

```diff
@@ -1039,6 +1039,10 @@ def validate(self, enforce_required=True, required_fields=None):
         ):
             errs['niceToHaveRequirements'] = 'answer_required'
 
+        if max_day_rate and 'dayRate' not in errs:
+            if float(self.data['dayRate']) > float(max_day_rate):
```

Comment:
Should we catch cast errors here?  Or we should fail because there's a real problem if dayRate isn't a number?